neuro_evolution_ctrnn/brain_visualizer/__init__.py

```diff
@@ -1,40 +1,260 @@
-import tkinter as tk
+from brain_visualizer.position import Positions
+from brain_visualizer.weights import Weights
+from brain_visualizer.neurons import Neurons
+from brain_visualizer.events import Events
+
+import json
+
+import pygame, sys
+from pygame.locals import *
 
 
 class BrainVisualizerHandler(object):
```

Comment:
Es ist nicht so üblich (glaube ich) in der __init__.py inhaltlichen Code zu haben. Die ist nur dafür da um imports in Python zu steuern. Siehe auch [diese Stackoverflow Frage](https://stackoverflow.com/a/47741568/11162327). Du könntest einfach eine brain_visualizer.py im selben Ordner erstellen und dort den Code reinpacken